CITATION

```diff
@@ -0,0 +1,5 @@
+@Software{RetDec,
```

Comment:
I think there was a misunderstanding. We would like to use `@Misc` here as `@Software` is only available if you use the [BibLaTeX](https://ctan.org/pkg/biblatex) package.